.gitignore

```diff
@@ -11,3 +11,7 @@ dist/
 
 # Ignore VSCode specifics
 .vscode
+
+# Ignore database manager js*
+database/dumps/meteor
+database/manager/*.js*
```

Comment:
One could consider to move this to a new `.gitignore` file under the database directory. That way, if we at some point change the script to another solution, e.g. docker compose setup. Then we have a logic grouping of all the necessary logic and config for the database  manager.
I would also say that it is the entire dumps folder that should be ignored :)

Do we generate js files for the database manager script?

Reply:
By ignoring a specific folder in dumps, I can deliver the folder (as structure in fs tree) upon checking out the code, and since is specifically in use for the purpose of data migration between the Mongo servers I treat it differently.

There are by-products of the ts script invocation (*.js, *.js.map).

Reply:
Is there an issue with the dumps folder not being in version control? As I see it, the dumps folder will be generated when I extract the data from the meteor database instance, and should only be created when that is done.

> There are by-products of the ts script invocation (*.js, *.js.map).
But we don't have a workflow, as far as I know, where these are generated.

Reply:
Will drop a tsconfig.json in folder to not emit nor create maps. Also dumop destination folder is automagically created, thus gitignoring it too.